src/main/events/library/open-game.ts

```diff
@@ -2,18 +2,28 @@ import { gameRepository } from "@main/repository";
 
 import { registerEvent } from "../register-event";
 import { shell } from "electron";
+import { spawn } from "child_process";
 import { parseExecutablePath } from "../helpers/parse-executable-path";
+import { parseLaunchOptions } from "../helpers/parse-launch-options";
 
 const openGame = async (
   _event: Electron.IpcMainInvokeEvent,
   gameId: number,
-  executablePath: string
+  executablePath: string,
+  launchOptions: string | null
 ) => {
   const parsedPath = parseExecutablePath(executablePath);
+  const parsedParams = parseLaunchOptions(launchOptions);
 
-  await gameRepository.update({ id: gameId }, { executablePath: parsedPath });
+  await gameRepository.update({ id: gameId }, { executablePath: parsedPath, launchOptions });
 
-  shell.openPath(parsedPath);
+  if (process.platform === "linux" || process.platform === "darwin") {
+    shell.openPath(parsedPath);
+  }
+
+  if (process.platform === "win32") {
+    spawn(parsedPath, parsedParams, { shell: false });
```

Comment:
```suggestion
    spawn(parsedPath, parsedParams, { shell: false, detached: true });
```
this should be enough